### PR TITLE
feat: Alert button created also refactored badge

### PR DESCRIPTION
**Pull request by ShoaibTamboli**
feat: Alert and button created. Also refactored CSS for badge and avatar.

### CODE REVIEW

File: components/Badge/badge.css [hunk 1]
```diff
@@ -12,19 +12,19 @@
...
 }
 
 .badge .offline{
-    background-color: hotpink;
+    background-color: var(--grey);
 }
 
 .badge .dnd{
-    background-color: greenyellow;
+    background-color: var(--danger-hover)
```
Comment (Bibin-Antony) [inline components/Badge/badge.css]: semicolon is missing

File: components/Alert/alert.html [hunk 2]
```diff
@@ -1,12 +1,240 @@
...
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Alert</title>
+    <link rel="stylesheet" href="/style.css">
+    <title>Mark-UI : Alert</title>
 </head>
+
 <body>
     this is alert page
```
Comment (sumitch29) [inline components/Alert/alert.html]: remove this

**Review by sumitch29 — APPROVED** [review]
Good to go.

File: components/Button/button.html [hunk 3]
```diff
@@ -0,0 +1,265 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (Bibin-Antony) [inline components/Button/button.html:7]: use meta tag description and author

File: components/Badge/badge.css (hunk 1 above)
Comment (ShoaibTamboli) [reply components/Badge/badge.css]: thanks, will update.

**Pull request merged**